.travis.yml

```diff
@@ -1,3 +1,5 @@
 language: node_js
 node_js:
-  - "10"
+  - '10'
+script:
+  - 'yarn verify'
```

Comment:
After some further deliberation, it has been decided better to follow a broader convention to use `yarn test` as our default therefore eliminating the need to specify the script here. (Use de-facto standards to minimize complexity so to speak)